doc/build-netbsd.md

```diff
@@ -60,7 +60,6 @@ With wallet:
 ./configure --with-gui=no CPPFLAGS="-I/usr/pkg/include" \
     LDFLAGS="-L/usr/pkg/lib" \
     BOOST_CPPFLAGS="-I/usr/pkg/include" \
-    BOOST_LDFLAGS="-L/usr/pkg/lib" \
```

Comment:
bitcoin#24524 can be done only after bitcoin#24301

Reply:
dropped